src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -521,13 +521,13 @@
 2 - Package ID
 3 - Package Sources</comment>
   </data>
-  <data name="PackageNamespaceMatchFound" xml:space="preserve">
-    <value>Package namespace matches found for package ID '{0}' are: '{1}'</value>
+  <data name="PackageSourceMappingPatternMatchFound" xml:space="preserve">
+    <value>Package source mapping pattern matches found for package ID '{0}' are: '{1}'</value>
     <comment>{0} - package id
 {1} - sources ids</comment>
   </data>
-  <data name="PackageNamespaceNoMatchFound" xml:space="preserve">
-    <value>Package namespace match not found for package ID '{0}'</value>
+  <data name="PackageSourceMappingPatternNoMatchFound" xml:space="preserve">
+    <value>Package source mapping pattern match not found for package ID '{0}'</value>
```

Comment:
If you agree with my feedback on removing pattern above, it'd apply here as well.

Reply:
removed pattern from the log message.